pkg/controller/gc.go

```diff
@@ -756,22 +756,11 @@ func (c *Controller) getVMLsps() []string {
 			vmLsp := ovs.PodNameToPortName(vm.Name, ns.Name, util.OvnProvider)
 			vmLsps = append(vmLsps, vmLsp)
 
-			attachNets, err := util.ParsePodNetworkAnnotation(vm.Spec.Template.ObjectMeta.Annotations[util.AttachmentNetworkAnnotation], vm.Namespace)
-			if err != nil {
```

Comment:
keep this section for  `util.AttachmentNetworkAnnotation` case

Reply:
This logic is not need for kubevirt. If you add this annotation, it is work only you not set vm multus network.